asapdiscovery-cli/asapdiscovery/cli/tests/test_meta_cli.py

```diff
@@ -34,6 +34,10 @@ def test_toplevel_runnable():
     ],
 )
 def test_subcommand_runnable(subcommand):
+
+    # xfail ml tests for now
+    if subcommand == "ml":
+        pytest.xfail("ML package is not currently available.")
```

Comment:
Instead of doing this, I think we could just do it directly in the parametrize. As in
```python
pytest.param("ml", marks=pytest.mark.xfail(reason="ML support is experimental.")
```
that is instead of `"ml"` directly in the paramters list. I hope that makes sense.

Reply:
More info here https://docs.pytest.org/en/stable/how-to/skipping.html#skip-xfail-with-parametrize